Tests/Testo/Invoice/ClientNote/ClientNoteServiceTest.php

```
@@ -0,0 +1,132 @@
+    private function formatDateNote(ClientNote $model): string
+    {
+        $dateNote = $model->getDateNote();
+        if (!$dateNote instanceof \DateTimeImmutable) {
+            throw new \RuntimeException('Expected DateTimeImmutable');
+        }
+        return $dateNote->format('Y-m-d');
+    }
+
+    private function makeService(
+        ?ClientNoteRepository $repository = null,
+        ?CR $cR = null,
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $cR. Configured minimum length is 3. (undefined)

(ShortVariable)